### PR TITLE
Fixes bug that stops me from messing with Olivia

### PR DESCRIPTION
**Pull request by mja00**
Weird bug was introduced, weird how it got there

### CODE REVIEW

File: src/main/java/pw/chew/chanserv/commands/ModeCommand.java [hunk 1]
```diff
@@ -47,10 +47,6 @@ protected void execute(SlashCommandEvent event) {
         Checks.notNull(user, "User");
         Checks.notNull(event.getGuild(), "Server");
 
-        if (MemberHelper.getRank(user).getPriority() >= MemberHelper.getRank(event.getMember()).getPriority()) {
-            event.replyEmbeds(ResponseHelper.generateFailureEmbed("Error Moment!", "You cannot change the mode of a user with a higher or equal rank.")).setEphemeral(true).queue();
-            return;
-        }
 
         if (mode.length() > 2) {
```
Comment (Chew) [inline src/main/java/pw/chew/chanserv/commands/ModeCommand.java]:
```suggestion
        if (mode.length() > 0.2) {
```

**Comment by Chew** [comment]
This PR has issues so I can't merge it, sorry!

**Comment by mja00** [comment]
WHAT

File: src/main/java/pw/chew/chanserv/commands/ModeCommand.java [hunk 2]
```diff
@@ -47,12 +47,8 @@ protected void execute(SlashCommandEvent event) {
         Checks.notNull(user, "User");
         Checks.notNull(event.getGuild(), "Server");
```
Comment (Chew) [inline src/main/java/pw/chew/chanserv/commands/ModeCommand.java]:
```suggestion
rbhueww
````

**Comment by Chew** [comment]
ok NOW it has issues sorry

**Comment by mja00** [comment]
Please stop spam committing my PR, I'll report your account.

**Pull request closed without merging**